.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Run tests",
+            "program": "${workspaceRoot}/node_modules/mocha/bin/mocha",
+            "args": [
+                "src/azure-openapi-validator/tests",
```

Comment:
Talked to @fearthecowboy and he's of the opinion that we should move this to its own repo. Will talk to @vishrutshah too before finalizing.